### ISSUE TITLE
Link that returns to currently locked tile

### ISSUE THREAD

**Issue by ethan-nelson** [hotosm/osm-tasking-manager2#381]
If you close the browser/tab that you have locked a tile on and reopen the TM, it would be great to have a button somewhere on the home or user page to "return to most recently locked tile".

**Comment by pellis2** [comment]
Agreed! After an edit session, if a user has not completed the cell but wants to return to edit it, it can be hard sometimes to find the cell again.
